### PR TITLE
chore(v2): e2e tests

### PR DESCRIPTION
**Pull request by scottgerring**
**Issue #, if available:**
#1562 

## Description of changes:

**Checklist**

 

* [ ] [Meet tenets criteria](https://docs.powertools.aws.dev/lambda-java/#tenets)
* [ ] Update tests
* [ ] Update docs
* [ ] PR title follows [conventional commit semantics](https://www.conventionalcommits.org/en/v1.0.0/)

## Breaking change checklist

**RFC issue #**:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jeromevdl — APPROVED** [review]

**Comment by jeromevdl** [comment]
e2e are still failing here...

File: examples/powertools-examples-idempotency/pom.xml [hunk 1]
```diff
@@ -95,7 +95,7 @@
                         </aspectLibrary>
                         <aspectLibrary>
                             <groupId>software.amazon.lambda</groupId>
-                            <artifactId>powertools-idempotency-dynamodb</artifactId>
```
Comment (scottgerring) [inline examples/powertools-examples-idempotency/pom.xml:98]: The aspect is in core, not DDB

File: powertools-e2e-tests/handlers/idempotency/pom.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
     <dependencies>
         <dependency>
             <groupId>software.amazon.lambda</groupId>
-            <artifactId>powertools-idempotency</artifactId>
+            <artifactId>powertools-idempotency-dynamodb</artifactId>
```
Comment (scottgerring) [inline powertools-e2e-tests/handlers/idempotency/pom.xml:18]: This was breaking tests

File: powertools-e2e-tests/handlers/idempotency/pom.xml [hunk 3]
```diff
@@ -43,7 +43,7 @@
                     <aspectLibraries>
                         <aspectLibrary>
                             <groupId>software.amazon.lambda</groupId>
-                            <artifactId>powertools-idempotency</artifactId>
+                            <artifactId>powertools-idempotency-core</artifactId>
```
Comment (scottgerring) [inline powertools-e2e-tests/handlers/idempotency/pom.xml:46]: this was breaking tests

File: powertools-e2e-tests/handlers/idempotency/src/main/java/software/amazon/lambda/powertools/e2e/Function.java [hunk 4]
```diff
@@ -27,7 +27,7 @@
 import software.amazon.lambda.powertools.idempotency.Idempotency;
 import software.amazon.lambda.powertools.idempotency.IdempotencyConfig;
 import software.amazon.lambda.powertools.idempotency.Idempotent;
-import software.amazon.lambda.powertools.idempotency.persistence.DynamoDBPersistenceStore;
+import software.amazon.lambda.powertools.idempotency.persistence.dynamodb.DynamoDBPersistenceStore;
```
Comment (scottgerring) [inline powertools-e2e-tests/handlers/idempotency/src/main/java/software/amazon/lambda/powertools/e2e/Function.java:30]: This was breaking tests

File: powertools-e2e-tests/handlers/largemessage_idempotent/pom.xml [hunk 5]
```diff
@@ -15,7 +15,7 @@
     <dependencies>
         <dependency>
             <groupId>software.amazon.lambda</groupId>
-            <artifactId>powertools-idempotency</artifactId>
+            <artifactId>powertools-idempotency-dynamodb</artifactId>
```
Comment (scottgerring) [inline powertools-e2e-tests/handlers/largemessage_idempotent/pom.xml:18]: This was breaking tests

File: powertools-e2e-tests/handlers/largemessage_idempotent/pom.xml [hunk 6]
```diff
@@ -47,7 +47,7 @@
                     <aspectLibraries>
                         <aspectLibrary>
                             <groupId>software.amazon.lambda</groupId>
-                            <artifactId>powertools-idempotency</artifactId>
+                            <artifactId>powertools-idempotency-core</artifactId>
```
Comment (scottgerring) [inline powertools-e2e-tests/handlers/largemessage_idempotent/pom.xml:50]: The aspect is in core, not DDB

**Pull request merged**